VCFv4.3.tex

```diff
@@ -370,6 +370,15 @@ \subsubsection{Fixed fields}
 	1000G		& 0		& Flag		& 1000 Genomes membership \\
 \end{longtable}
 
+\begin{itemize}
+\renewcommand{\labelitemii}{$\circ$}
+\item END: End reference position, indicating that the variant spans positions POS--END on reference/contig CHROM.
+Normally this is the position of the last base in the REF allele, so can be derived from POS and the length of REF, and no END INFO field is needed.
```

Comment:
missing it?
```suggestion
Normally this is the position of the last base in the REF allele, so it can be derived from POS and the length of REF, and no END INFO field is needed.
```

Reply:
Intentional without it, having the “this” clause as the subject. But maybe adding “it” improves it given the overabundance of commas needed for clarity…

Reply:
Improved by your suggestion, thanks.